packages/core/src/app/cart/mapToOrderSummarySubtotalsProps.ts

```diff
@@ -14,15 +14,39 @@ export default function mapToOrderSummarySubtotalsProps({
     coupons,
     taxes,
     fees,
-}: Checkout): OrderSummarySubtotalsProps {
+}: Checkout,
+isShippingDiscountDisplayEnabled: boolean,
+): OrderSummarySubtotalsProps {
+    const allConsignmentsHaveSelectedShippingOption = hasSelectedShippingOptions(consignments);
+
+    if (!isShippingDiscountDisplayEnabled) {
```

Comment:
Can we use control flow to determine the impacted values instead of using the condition to early return an almost same object?